Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerTreeView.cpp

```diff
@@ -228,49 +252,109 @@ namespace AzToolsFramework
             return;
         }
 
-        //if we are attempting to drag an unselected item then we must special case drag and drop logic
-        //QAbstractItemView::startDrag only supports selected items
-        if (m_queuedMouseEvent)
+        // If the index is selected, we should move the whole selection.
+        if (selectionModel()->isSelected(indexAtClick))
         {
-            if (!index.isValid() || index.column() != 0)
-            {
-                return;
-            }
+            StartCustomDrag(selectionModel()->selectedIndexes(), defaultDropAction());
+        }
+        else
+        {
+            StartCustomDrag({ indexAtClick }, defaultDropAction());
```

Comment:
suggestion/nit: Including the type here before the `{` is actually really useful for maintenance (find usages etc..) so I'd recommend adding it even though it's not strictly required